### ISSUE TITLE
Move out application specific storage

### ISSUE THREAD

**Issue by JeroenDeDauw** [wmde/FundraisingStore#116]
A step towards https://github.com/wmde/FundraisingStore/issues/115

Database tables only used by one application don't need to be in this shared component. (The same goes for fields, though that is definitely more hairy to refactor.)

Can be moved to the management app:

* BackendBanner.php
* BackendImpression.php
* User.php
* ActionLog.php?
* Address.php?

**Comment by manicki** [comment]
+1

> ActionLog.php?

AFAIR this one is "operations-center" specific too, so should be also moved out.

> Address.php?

this in turn seems as "shared" as Donation etc are

**Comment by JeroenDeDauw** [comment]
Yeah, checked Address now with my IDE, and it's part of Subscription, directly bound to in the Fun app even outside its data access layer #grump
